test/test_jit.py

```diff
@@ -4062,6 +4062,18 @@ def forward(self, x, w):
         with self.assertRaisesRegex(RuntimeError, 'test_jit.py\", line {}'.format(lineno + 3)):
             loaded(torch.rand(3, 4), torch.rand(30, 40))
 
+    def test_ignorable_args(self):
```

Comment:
Generally we should avoid adding more tests to `test_jit.py`, which is growing too large to be manageable. Is there another test file that it fits in? If not, should we create one?